sources/install/package_web.sh

```diff
@@ -894,6 +894,19 @@ function install_postman() {
     add-history postman
     add-test-command "which postman"
     add-to-list "postman,https://www.postman.com/,API platform for testing APIs"
+
+function install_zap() {
+    colorecho "Installing ZAP"
+    mkdir /opt/tools/ZAP
+    # using $(which curl) to avoid having additional logs put in curl output being executed because of catch_and_retry
+    zap_version=$(curl -s "https://www.zaproxy.org/download/" | grep -E -o "ZAP [0-9]+(\.[0-9]+)+" | head -1 | cut -d' ' -f2)
+    wget "https://github.com/zaproxy/zaproxy/releases/download/v${zap_version}/ZAP_${zap_version}_Linux.tar.gz" -O /tmp/zap.tar.gz
+    tar -xf /tmp/zap.tar.gz -C /opt/tools/
+    mv /opt/tools/ZAP_${zap_version} /opt/tools/ZAP/
```

Comment:
`mv /tmp/ZAP_${zap_version}/* /opt/tools/ZAP/`

Reply:
@noahfraiture,

You must put /tmp/…/* to move the contents of the folder into the /opt/tools/ZAP folder